npm/react18/.npmignore

```diff
@@ -0,0 +1,3 @@
+examples
```

Comment:
Do we even need this file? Looks like [files](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#files) accomplishes the same.

Reply:
gone